### ISSUE TITLE
Linux build fail: 'const class testing::TestInfo' has no member named 'test_suite_name'

### ISSUE THREAD

**Issue by sungminlee114** [colmap/colmap#2031]
While building from source, ninja fail due to 

'const class testing::TestInfo' has no member named 'test_suite_name' at colmap/util/testing.cc

Maybe it's due to https://github.com/colmap/colmap/commit/8af292cb096b9478703821aa2e84730145b203a1

**Comment by ahojnnes** [comment]
Which great version are you using? Which OS?

**Comment by sungminlee114** [comment]
I'm using version 3.8, the latest commit in dev branch. My OS is debian docker image

**Comment by sungminlee114** [comment]
It turned out to be gtest installation error. 

Anyone facing same errors can try https://github.com/google/googletest/blob/main/googletest/README.md.

**Comment by sijieaaa** [comment]
> It turned out to be gtest installation error.
> 
> Anyone facing same errors can try https://github.com/google/googletest/blob/main/googletest/README.md.

May I know how to solve this issue?

**Comment by sungminlee114** [comment]
@sijieaaa 

TL;DR
1. The error is due to either misinstallation or version mismatch of `gtest`.
2. Among many of installation guides I've been through, kind instructions written in [this link](https://github.com/google/googletest/blob/main/googletest/README.md#standalone-cmake-project) did the work.
3. So what you should do is (maybe purge you're pre-installed `gtest` module and) visit the link and follow the instructions to build the `gtest` module in your environment.

Hope this helps
